internal/security-governance-system/webapp/pom.xml

```diff
@@ -60,16 +60,38 @@
             <artifactId>spring-retry</artifactId>
             <version>1.2.1.RELEASE</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-yaml</artifactId>
+            <version>2.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.7.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.3</version>
+        </dependency>
+        <!-- Apache HttpClient Mime -->
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpmime</artifactId>
+            <version>4.5.3</version>
+        </dependency>
     </dependencies>
     <build>
-        <finalName>automation-manager-webapp</finalName>
+        <finalName>scan-manager-webapp</finalName>
         <plugins>
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
                     <fork>true</fork>
-                    <mainClass>org.wso2.security.tools.am.webapp.Application</mainClass>
+                    <mainClass>org.wso2.security.tools.sgs.webapp.Application</mainClass>
+                    <jvmArguments>-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005</jvmArguments>
```

Comment:
Do we need to keep debug enabled? This is a security threat if used incorrectly in production.